Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/debuglog/DebugLogger.kt

```diff
@@ -44,8 +45,19 @@ object DebugLogger : DebugLoggerBase() {
         context = application
 
         try {
-            if (triggerFile.exists()) {
-                Timber.tag(TAG).i("Trigger file exists, starting debug log.")
+            val startLogger = when {
+                triggerFile.exists() -> {
+                    Timber.tag(TAG).i("Trigger file exists, starting debug log.")
+                    true
+                }
+                CWADebug.isDeviceForTestersBuild -> {
```

Comment:
Now we don't have the option anymore to disable logging on deviceForTestersBuild. Every restart will overwrite the last setting and automatically turn on logging. Might be problematic for low device storage tests or alike.

Reply:
Has this been the case yet? Even low end devices have a few 100MB storage. Test devices often have no other apps installed. I'd say we fix this it becomes an issue for testers.

Reply:
Just my small 2 cents for future enhancement:
What about letting the logger for production and DeviceForTesters builds query the available disk space from time to time, and automatically stop logging when it gets below a threshold of 100MB?